src/NumericalAlgorithms/Spectral/Projection.hpp

```diff
@@ -17,6 +17,9 @@ namespace Spectral {
 /// The portion of an element covered by a mortar.
 enum class MortarSize { Full, UpperHalf, LowerHalf };
 
+std::ostream& operator<<(std::ostream& os,
+                         const MortarSize& mortar_size) noexcept;
```

Comment:
why not take by value? enum`s are usually smaller than the pointer that the reference creates

Reply:
I'll squash this ok

Reply:
of course!